letsencrypt/cli.py

```diff
@@ -934,6 +1011,8 @@ def _handle_exception(exc_type, exc_value, trace, args):
             traceback.format_exception(exc_type, exc_value, trace)))
 
 
+# this copy of plugins can be mocked out
+plugins_testable = plugins_disco.PluginsRegistry.find_all
```

Comment:
this should be removed in favor of mocking out `plugins_disco.PluginsRegistry.find_all`